rotkehlchen/db/filtering.py

```diff
@@ -545,12 +545,13 @@ class DBEqualsFilter(DBFilter):
     column: str
     value: Union[str, bytes, int, bool]
     alias: Optional[str] = None
+    operator: str = '='
```

Comment:
Hmm I don't really like this much. The filter is called `DBEquals`. So I would expect either `=` or `!=`. Now you are mixing in the `IS` and `IS NOT` which is already used in the `DBNullFilter`.

Perhaps you can turn it into a `DBComparisonFilter` and make it parametrizable with operator and value and then use it to replace all other uses of filters which can be parametrized and replaced?